modules/perception/onboard/component/fusion_camera_detection_component.cc

```diff
@@ -143,7 +143,7 @@ bool GetProjectMatrix(
     const std::map<std::string, Eigen::Matrix4d> &extrinsic_map,
     const std::map<std::string, Eigen::Matrix3f> &intrinsic_map,
     Eigen::Matrix3d *project_matrix, double *pitch_diff = nullptr) {
-  if (camera_names.size() != 2) {
+  if (camera_names.size() != 2) {  // To Do: This condition should be removed.
```

Comment:
use TODO (techoe) for better tracking and list it as a separate line of code.

Reply:
Thank you for the review! It was fixed.